tests/queries/0_stateless/01615_random_one_shard_insertion.sql

```diff
@@ -1,22 +1,26 @@
-drop table if exists shard;
+create database if not exists shard_0;
+create database if not exists shard_1;
+drop table if exists shard_0.tbl;
+drop table if exists shard_1.tbl;
 drop table if exists distr;
 
-create table shard (id Int32) engine = MergeTree order by cityHash64(id);
-create table distr as shard engine Distributed (test_cluster_two_shards_localhost, currentDatabase(), shard);
-
-insert into distr (id) values (0), (1);  -- { serverError 55; }
+create table shard_0.tbl (number UInt64) engine = MergeTree order by number;
+create table shard_1.tbl (number UInt64) engine = MergeTree order by number;
+create table distr (number UInt64) engine = Distributed(test_cluster_two_shards_different_databases, '', tbl);
```

Comment:
If this test uses test_cluster_two_shards_different_databases cluster, then it is better to mark it as not suitable for parallel run (https://github.com/ClickHouse/ClickHouse/blob/82b6f49516c0cce17d28c6d70202f634492d830a/tests/queries/skip_list.json#L399)

Reply:
done